src/index/src/inverted_index/format.rs

```diff
@@ -27,10 +27,10 @@
 //!
 //! An inverted index comprises a collection of bitmaps, a null bitmap, and a finite state transducer (FST) indicating tag values' positions:
 //!
-//! `bitmap₀ bitmap₁ bitmap₂ ... bitmapₙ null_bitmap fst`
+//! `null_bitmap bitmap₀ bitmap₁ bitmap₂ ... bitmapₙ fst`
```

Comment:
Looks like we move forward the null_bitmap before bitmaps, can you explain why?

Reply:
1. For Read, bitmap reading relies on offset and size. Whether `null_bitmap` comes first or not does not affect the format and will not destroy the implementation.
2. For Write, some implementations of index building may directly use `BTreeMap<Option<Bytes>, BitVec>>`. In this case, it is more natural to write `null_bitmap` first. (Of course, this is not the case in my implementation, but I'm happy to handle nulls first)

Reply:
Thank you for the info.